src/main/flight/gps_rescue.c

```diff
@@ -733,7 +733,7 @@ void descend(void)
 
     // descend faster while the quad is at higher altitudes
     const float descentRateMultiplier = constrainf(rescueState.intent.targetAltitudeCm / 5000.0f, 0.0f, 1.0f);
-    altitudeStep *= 1.0f + (2.0f * descentRateMultiplier); 
```

Comment:
(BTW: some of these whitespace only changes are causing merge conflicts. Separate PR would be better)

Reply:
Reintroduced the whitespaces